src/index.ts

```diff
@@ -1,19 +1,35 @@
 import express, { Express, Response, Request } from "express";
+import mongoose, { ConnectOptions } from "mongoose";
 import dotenv from "dotenv";
 
+// Initialize configuration from .env file
 dotenv.config();
 
+// Express app setup
 const app: Express = express();
-const port = process.env.PORT || 3000;
+const port = process.env.PORT || 3000; // Server port
 
+// Database connection
+const databaseUrl = process.env.DATABASE_URL || ""; // Ensure DATABASE_URL is defined
+mongoose
+  .connect(databaseUrl, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  } as ConnectOptions)
+  .then(() => console.log("Connected to MongoDB Atlas"))
+  .catch((err) => console.error("Could not connect to MongoDB Atlas", err));
+
+// Basic route for home page
 app.get("/", (_, res: Response) => {
   res.send("Hello World!");
 });
 
+// Echo function to display request URLs
 function echo(req: Request, res: Response) {
   res.send(req.originalUrl);
 }
 
+// Additional routes for bathroom finder functionality
```

Comment:
these are the routes that I specified in a previous task. please implement these routes rather than creating your own. not necessary to add any routes further than these, as these are the routes that we will need for all P0 functionality. please see [this issue](https://github.com/Green-Kittens/bathroom-finder/issues/38) for more context